content/articles/choosing-wagtail.md

```diff
@@ -0,0 +1,404 @@
+[download the slide here](/docs/wagtailspace-20180614.pdf), in PDF format.
+
+We the DevTeam at the CTL continuously research and evaluate existing and new
+software for possible uses in developing educational technologies at Columbia
+University. In building tools and digital environments for classrooms and
+online learning, our approach is grounded in the
+[CTL’s mission](https://ctl.columbia.edu/about/),
+which is to support and promote the _practice_ and _culture_ of _teaching_ and
+_learning_.
+
+Our projects always begin with an educational hypothesis, one that imagines
+ways that software or digital tools can promote learner-centered objectives. We
+help faculty in supporting the tools they choose, and also explore the
+frontiers of educational technologies that engages students towards better
+scholarship. Our research can also lead us to create novel solutions for
+these pedagogical goals.
+
+We work closely with the faculty to understand the learner-centered objectives
+within the projects, and from there, we use the goals as the primary basis of
+the user stories. Based on the sets of user stories, we recommend the best
+approach to adopt. These are three of the most common approaches we proposed to
+the faculty clients:
+
+<ol style="list-style: none; margin: 0; padding: 0;">
+	<li style="background: url('/img/assets/icon-tool.png') no-repeat left 15px; min-height: 50px; padding: 10px 0 10px 65px;"><i>Custom web-based applications:</i></br />
+	We develop these tools as solutions to complex learning environment.<br />
+	Examples: <a href="http://mediathread.info">Mediathread</a>, <a href="https://econpractice.ctl.columbia.edu">EconPractice</a>, <a href="https://writlarge.ctl.columbia.edu">Writ Large</a></li>
+	<li style="background: url('/img/assets/icon-static.png') no-repeat left 15px; min-height: 50px; padding: 10px 0 10px 65px;"><i>“Static” websites:</i><br />
+	These sites hold content that doesn’t have frequent editing cycle, and has simple data model.<br />
+	Examples: <a href="https://filmglossary.ccnmtl.columbia.edu">The Film Language Glossary</a></li>
+	<li style="background: url('/img/assets/icon-hybrid.png') no-repeat left 15px; min-height: 50px; padding: 10px 0 20px 65px;"><i>Hybrid solutions:</i><br />
+	The environment of this type has content provided and edited by faculty clients, and tools developed and maintained by CTL.<br />
+	Example: <a href="https://pass.ctl.columbia.edu">PASS</a> and <a href="https://match.ctl.columbia.edu">MATCH</a></li>
+</ol>
+
+Our primary framework at CTL is Python-based
+[Django](https://www.djangoproject.com/) and it works very well in our
+development of custom web-based applications. However, Django admin UI is not
+quite a CMS UI that can be used easily by content managers and providers.
+
+## The need for a CMS
+
+For the projects that fall under “static website” or “hybrid solution”, faculty
+clients work closely with us as content providers, and would be editing and
```

Comment:
its unclear who the content provider is, "faculty" or "us"

Reply:
Fixed